src/protobuf_reified.ml

```diff
@@ -76,11 +76,11 @@ struct
     field_type: field;
     field_class: [`required | `optional of string option | `repeated]
   }
-  and method_ = {
-    method_name: string;
-    input_type: string;
-    output_type: string;
-  }
+  (* and method_ = {
+   *   method_name: string;
+   *   input_type: string;
+   *   output_type: string;
+   * } *)
   and basic = {
```

Comment:
Why is `method_` commented out?  If it's due to a warning arising from `dune`'s defaults, let's disable the warning locally instead.